draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3384,58 +3385,98 @@ dCDN MAY reject such requests for urgent processing using “ereject” error co
    }
  ]
 }
-        ]]></artwork>
-          </figure>
-        </section>
-      </section>
+        ]]></artwork></figure>
+        	</section>
+      	</section>
 
 
-       
-
-            <section anchor="object-list-type-capability-object" title="CI/T Object List Type Capability Object">
+	<section anchor="object-list-type-capability-object" title="CI/T Object List Type Capability Object">
         <t>
-           Given an object list being supported by the dCDN, the CI/T Object List Type capability object is used to indicate support for one or more
-           Object List types listed in <xref target="IANA.CDNI.ObjectListTypeReg"/> by the type property of the "ObjectList" object. Capability type is  "FCI.TriggerObjectListType".
+		Given an object list being supported by the dCDN, the CI/T Object List Type capability object is used to 
+		indicate support for one or more Object List types listed in <xref target="IANA.CDNI.ObjectListTypeReg"/> 
+		by the type property of the "ObjectList" object. The capability type is "FCI.CITObjectListType".
         </t>
+        <t><list style="empty" hangIndent="3">
+	<t><list style="hanging" hangIndent="3">
 
-        <t><list style="empty">
-           <t>Property: object-list-types<list style="empty">
+		<t hangText="Name: object-list-types">
+		<vspace blankLines="1"/>
+		Description: A list of supported object list types.
+		<vspace blankLines="1"/>
+		Value: An array <xref target="object-list-type"/>.
+		<vspace blankLines="1"/>
+		Mandatory: Yes.
```

Comment:
Looks like Mandatory changed here.  What should happen if the CDN supports no object list types?

Reply:
Added a standard comment about missing or empty list amounts to no support